Build Error

error after `make tools`

```
error[E0599]: no variant or associated item named `XDai` found for enum `ethers_core::types::Chain` in the current scope
  --> /Users/mac/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ethers-etherscan-2.0.10/src/lib.rs:92:20
   |
92 |             Chain::XDai |
   |                    ^^^^ variant or associated item not found in `Chain`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `ethers-etherscan` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error[E0425]: cannot find value `XDai` in this scope
  --> /Users/mac/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ethers-contract-2.0.10/src/multicall/constants.rs:33:9
   |
33 |         XDai as u64,                     // Gnosis Chain
   |         ^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `ethers-contract` (lib) due to previous error
error: failed to compile `create3 v0.1.0 (https://github.com/darwinia-network/create3-deploy#398cd35d)`, intermediate artifacts can be found at `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installTBHCTL`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
make: *** [create3] Error 101

```

```
   Compiling ethers-signers v2.0.10
error: failed to run custom build command for `svm-rs-builds v0.2.0`

Caused by:
  process didn't exit successfully: `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo/release/build/svm-rs-builds-4bffe3450c1e65cc/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /Users/mac/.cargo/registry/src/index.crates.io-6f17d22bba15001f/svm-rs-builds-0.2.0/build.rs:140:46:
  Failed to fetch releases: ReqwestError(reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("github.com")), port: None, path: "/alloy-rs/solc-builds/raw/55ef6c89b8a09c16feee4a424f8a53f822831a61/macosx/aarch64/list.json", query: None, fragment: None }, source: hyper::Error(Connect, Os { code: 54, kind: ConnectionReset, message: "Connection reset by peer" }) })
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `create3 v0.1.0 (https://github.com/darwinia-network/create3-deploy#398cd35d)`, intermediate artifacts can be found at `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
make: *** [create3] Error 101

```

A new type of error, I consider this as a progress XD

> ```
>    Compiling ethers-signers v2.0.10
> error: failed to run custom build command for `svm-rs-builds v0.2.0`
> 
> Caused by:
>   process didn't exit successfully: `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo/release/build/svm-rs-builds-4bffe3450c1e65cc/build-script-build` (exit status: 101)
>   --- stderr
>   thread 'main' panicked at /Users/mac/.cargo/registry/src/index.crates.io-6f17d22bba15001f/svm-rs-builds-0.2.0/build.rs:140:46:
>   Failed to fetch releases: ReqwestError(reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("github.com")), port: None, path: "/alloy-rs/solc-builds/raw/55ef6c89b8a09c16feee4a424f8a53f822831a61/macosx/aarch64/list.json", query: None, fragment: None }, source: hyper::Error(Connect, Os { code: 54, kind: ConnectionReset, message: "Connection reset by peer" }) })
>   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> warning: build failed, waiting for other jobs to finish...
> error: failed to compile `create3 v0.1.0 (https://github.com/darwinia-network/create3-deploy#398cd35d)`, intermediate artifacts can be found at `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo`.
> To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
> make: *** [create3] Error 101
> ```
> 
> A new type of error, I consider this as a progress XD

It seems a Network error.

> > ```
> >    Compiling ethers-signers v2.0.10
> > error: failed to run custom build command for `svm-rs-builds v0.2.0`
> > 
> > Caused by:
> >   process didn't exit successfully: `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo/release/build/svm-rs-builds-4bffe3450c1e65cc/build-script-build` (exit status: 101)
> >   --- stderr
> >   thread 'main' panicked at /Users/mac/.cargo/registry/src/index.crates.io-6f17d22bba15001f/svm-rs-builds-0.2.0/build.rs:140:46:
> >   Failed to fetch releases: ReqwestError(reqwest::Error { kind: Request, url: Url { scheme: "https", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("github.com")), port: None, path: "/alloy-rs/solc-builds/raw/55ef6c89b8a09c16feee4a424f8a53f822831a61/macosx/aarch64/list.json", query: None, fragment: None }, source: hyper::Error(Connect, Os { code: 54, kind: ConnectionReset, message: "Connection reset by peer" }) })
> >   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> > warning: build failed, waiting for other jobs to finish...
> > error: failed to compile `create3 v0.1.0 (https://github.com/darwinia-network/create3-deploy#398cd35d)`, intermediate artifacts can be found at `/var/folders/3f/gp4jvny97d7fnr62w94th7m80000gn/T/cargo-installdpK8eo`.
> > To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
> > make: *** [create3] Error 101
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > A new type of error, I consider this as a progress XD
> 
> It seems a Network error.

Worked like a breeze after I change VPN, sorry I should AI first

wondering how u resolve this XD

--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ salt   :; @create3 -s 00000000000000
 sync   :; @git submodule update --recursive
 
 tools  :  foundry create3
-create3:; @cargo install --git https://github.com/darwinia-network/create3-deploy -f
+create3:; @cargo install --git https://github.com/darwinia-network/create3-deploy -f --locked
 foundry:; curl -L https://foundry.paradigm.xyz | bash
 
 deploy :; @./bin/deploy.sh

